allauth/socialaccount/providers/kakao/provider.py

```diff
@@ -25,10 +25,9 @@ def extract_uid(self, data):
         return str(data['id'])
 
     def extract_common_fields(self, data):
-        email = data['kakao_account'].get('email')
         nickname = data['properties'].get('nickname')
 
-        return dict(email=email, username=nickname)
+        return dict(username=nickname)
```

Comment:
You mention: "Kakao no longer offers email as a required field" -- but the existing code does not require the email to be present. If it is not there, email=None is passed so everything will work as intended.